mpi-proxy-split/Makefile

```diff
@@ -128,7 +128,9 @@ tidy:
 clean: tidy
 	rm -f ${LIBOBJS} ${MANA_COORD_OBJS}
 	rm -f libmana.so
-	rm -f ${MANA_ROOT}/lib/dmtcp/libmana.so
+	if test -d ${MANA_ROOT}/lib/dmtcp/libmana.so; then \
```

Comment:
`-d` is for directory and `libmana.so` is a file, not directory.

Reply:
and rm -f  works whether you the file exists or not. I don't think you need to test anything here.